config.sample.yaml

```
@@ -182,6 +182,9 @@ ircService:
         # HS attached to this AS will be able to interact with this room.
         # Default: true.
         federate: true
+        # Force this room version when creating IRC channels. Beware if the homeserver doesn't
+        # support the room version then the request will fail. By default, no version is requested.
```

We should clarify what this means:
```suggestion
        # support the room version then the request will fail. By default, the bridge will use whatever the homeserver's default is.
```